sys/userlib/src/kipc.rs

```diff
@@ -24,6 +26,20 @@ pub fn read_task_status(task: usize) -> abi::TaskState {
     ssmarshal::deserialize(&response[..len]).unwrap_lite().0
 }
 
+pub fn find_faulted_task(task: usize) -> Option<NonZeroUsize> {
```

Comment:
Perhaps this ought to have a doc comment? I know there's docs for the KIPC interface in the Hubris reference, but might we want to include the same documentation in the actual Rust API?

Reply:
In fairness, it looks like the rest of `userlib::kipc` lacks them...except for the one I added ;-)

https://github.com/oxidecomputer/hubris/blob/d6a6e67e2377a986ea78006324c108cf55154f8a/sys/userlib/src/kipc.rs#L130-L131

Reply:
No reason not to improve things while I'm in here!